archunit/src/main/java/com/tngtech/archunit/core/importer/JavaClassProcessor.java

```diff
@@ -327,6 +328,19 @@ public void visitFieldInsn(int opcode, String owner, String name, String desc) {
         public void visitMethodInsn(int opcode, String owner, String name, String desc, boolean itf) {
             accessHandler.handleMethodInstruction(owner, name, desc);
         }
+        
+        @Override
+		public void visitInvokeDynamicInsn(String name, String descriptor, Handle bootstrapMethodHandle,
+				Object... bootstrapMethodArguments) {
+			for (Object o : bootstrapMethodArguments) {
+				if (o instanceof Handle) {
+					Handle currentHandle = (Handle) o;
+					accessHandler.handleMethodInstruction(currentHandle.getOwner(), currentHandle.getName(),
+							currentHandle.getDesc());
+					break;
```

Comment:
Can we break the loop because `bootstrapMethodArguments` cannot contain more than one `Handle`?
(I must admit that I'm not familiar with `visitInvokeDynamicInsn` at all, I've so far only seen [ASM's javadoc](https://asm.ow2.io/javadoc/org/objectweb/asm/MethodVisitor.html#visitInvokeDynamicInsn-java.lang.String-java.lang.String-org.objectweb.asm.Handle-java.lang.Object...-)...)